torch/csrc/jit/script/compiler.cpp

```diff
@@ -3193,8 +3193,14 @@ std::unique_ptr<Function> CompilationUnit::define(
   auto creator = [def, _resolver, self](Function& method) {
     // Store the function name so that it can be referenced if there is an error
     // while compiling this function
-    ErrorReport::CallStack call(
-        self ? method.qualname().qualifiedName() : method.qualname().name());
+    std::string call_name = method.qualname().name();
+    if (self) {
+      auto atoms = method.qualname().atoms();
```

Comment:
This is removing all of the qualified name?

Reply:
Only the parts that are not `ClassName.method_name`, they're mostly visual noise